languages/csharp/config.json

```
@@ -24,6 +24,12 @@
         "concepts": ["booleans"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "chars",
+        "uuid": "e74c6cd6-add7-45d1-abcb-513418e9d2d0",
+        "concepts": ["chars", "StringBuilder"],
+        "prerequisites": ["for-loops", "strings"]
```

I think `conditionals` should also be added as the student is expected to write `if` statements:

```suggestion
        "prerequisites": ["conditionals", "for-loops", "strings"]
```